lvgl-sys/build.rs

```diff
@@ -156,17 +121,27 @@ fn main() {
 
     cfg.compile("lvgl");
 
+    #[cfg(feature = "drivers")]
+    link_extra.split(',').for_each(|a| {
+        println!("cargo:rustc-link-lib={a}");
+        //println!("cargo:rustc-link-search=")
+    });
+}
+fn generate_bindings(conf: &BuildConf) {
     let mut cc_args = vec![
         "-DLV_CONF_INCLUDE_SIMPLE=1",
         "-I",
-        lv_config_dir.to_str().unwrap(),
+        conf.lv_config_dir.to_str().unwrap(),
         "-I",
-        vendor.to_str().unwrap(),
+        conf.vendor.to_str().unwrap(),
         "-fvisibility=default",
     ];
 
     // Set correct target triple for bindgen when cross-compiling
-    let target = env::var("TARGET").expect("Cargo build scripts always have TARGET");
+    let target = env::var("CROSS_COMPILE").map_or_else(
+        |_| env::var("TARGET").expect("Cargo build scripts always have TARGET"),
+        |c| c.trim_end_matches('-').to_owned(),
```

Comment:
This may be set incorrectly or by some other process (or accidentally left in a shell); may be worth falling back to the `TARGET` var if compile fails with this and warn